modules/claims_api/spec/requests/v2/veterans/claims_request_spec.rb

```diff
@@ -169,10 +169,10 @@
         end
 
         it 'are listed' do
-          lighthouse_claim = create(:auto_established_claim, status: 'PEND', veteran_icn: veteran_id,
+          lighthouse_claim = create(:auto_established_claim, status: 'established', veteran_icn: veteran_id,
                                                              evss_id: '600098193')
-          lighthouse_claim_two = create(:auto_established_claim, status: 'CAN', veteran_icn: veteran_id,
-                                                                 evss_id: '600098194')
+          lighthouse_claim_two = create(:auto_established_claim, status: 'pending', veteran_icn: veteran_id,
+                                                                 evss_id: nil)
```

Comment:
lighthouse claims can only have the status [shown here](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/claims_api/app/models/claims_api/auto_established_claim.rb#L31-L35). The previous scenarios were not possible. evss_id set to nil in :175 for 'pending' status as it would not have been submitted upstream yet (successfully)